actions/Import.php

```diff
@@ -23,37 +23,38 @@
 
 namespace oat\taoTestTaker\actions;
 
-use common_report_Report;
-use core_kernel_classes_Resource;
-use oat\generis\Helper\UserHashForEncryption;
-use oat\generis\model\GenerisRdf;
 use oat\taoTestTaker\models\CsvImporter;
-use oat\taoTestTaker\models\events\TestTakerImportedEvent;
-use oat\taoTestTaker\models\TestTakerSavePasswordInMemory;
-use tao_helpers_form_FormFactory;
+use oat\taoTestTaker\models\RdfImporter;
+use tao_actions_Import;
```

Comment:
can we create an alias or something that is more consistent in the import

Reply:
I think we must agree on it with the team. I personally do not use alias unless the class name is overriding other, cause it makes difficult to differentiate we are dealing with legacy namespace or new ones. I find it more confuse than good. IMO.

Reply:
I knid of agree with @gabrielfs7 , a new implementation is always better. But I also agree with @mike-ionut-mihai-sandu-tao because in case of namespacification, we do not need to find all use cases (only import). Need to discuss in a team

Reply:
Can I skip this till we have an agreement?